{"name": "test_login_1_error[data1]", "status": "passed", "parameters": [{"name": "data", "value": "{'user': '135', 'pwd': '', 'expected': '请输入正确的手机号'}"}], "start": 1569945388449, "stop": 1569945388821, "uuid": "6b0d67f3-da6d-41e8-8beb-d0fca620cb8f", "historyId": "f171d146038a07fdfeef2c51f1b1038a", "testCaseId": "0754122d22300ba419cd91efb872670c", "fullName": "test_cases.test_login.TestLogin#test_login_1_error", "labels": [{"name": "tag", "value": "error"}, {"name": "parentSuite", "value": "test_cases"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-I3J3MGM"}, {"name": "thread", "value": "528-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_cases.test_login"}]}